src/Utility/TypeExtensions.cs

```diff
@@ -65,7 +65,7 @@ private static HttpRequestContext ToHttpRequestContext (this RpcHttp rpcHttp)
             return httpRequestContext;
         }
 
-        internal static object ToObject(this TypedData data, bool convertFromJsonIfValidJson = true)
+        internal static object ToObject(this TypedData data, bool convertFromJsonIfValidJson = true, bool isDurableClient = false)
```

Comment:
I wonder if we want to rename the parameter `isDurableClient` to something more general like `makeCaseInsensitive`. I think that tells us more about what the output of this method will be (a case sensitive or insensitive object) whereas `isDurableClient` seems overly specific to the usecase. Just a suggestion!

Reply:
Fixed. The new parameter name is `returnCaseInsensitiveHashtable`.